src/pages/Consumer/Create.tsx

```diff
@@ -68,8 +68,18 @@ const Page: React.FC = (props) => {
         setStep(nextStep);
       });
     } else if (nextStep === 3) {
-      const isValid = Object.keys(plugins).some((name) => name.includes('auth'));
-      if (!isValid) {
+      const authPluginNames = [
```

Comment:
oh my mistake, you could import an plugin array from https://github.com/api7/dashboard-components/blob/master/packages/plugin/src/data.tsx#L4

Reply:
```js
import {PLUGIN_MAPPER_SOURCE} from '@api7-dashboard/plugin;
```

https://github.com/api7/dashboard-components/blob/master/packages/plugin/src/data.tsx#L37

Reply:
got it~thanks, already done 😊